### DISCUSSION TITLE
move to the last tab in a browser window!

### DISCUSSION THREAD

**Discussion by nivethan-me** [gdh1995/vimium-c#670] [Q&A] [answered]
hi, is there any way to move to the last tab in a window with a shortcut?

**Comment by gdh1995** [comment, marked as answer]
There has been a mapping of `g$` (to the `lastTab` command).

BTW, because most commands support count prefixes, so you may press `999K` (aka `999gt`) to go to the 999-th tab on the right, and then if only there're less than 500 tabs in a current window, Vimium C's `nextTab` will not go back to a first tab, but go to a last tab directly.

Then you may use such a map to make "count prefix" be 999 by default:
```
# usually it works just like `map g9 lastTab`
map g9 nextTab count=9999
```
